ngraph/test/type_prop/dft.cpp

```diff
@@ -0,0 +1,285 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/type_prop.hpp"
+
+using namespace ngraph;
+
+TEST(type_prop, dft_constant_axes_and_there_are_no_signal_size_static_shapes)
+{
+    struct ShapesAndValues
```

Comment:
Why do we need to have `ShapesAndValues` struct inside the body of tests?

Reply:
Done.